entry_tasks/16_Pick_movie_by _genre_or_actor/main.py

```diff
@@ -0,0 +1,29 @@
+if __name__ == "__main__":
+    a = input("Start searching by genre: ").strip()
+    genres_data = {
+        "comedy": ["Meet the Parents", "Anger Management"],
+        "adventures": ["Mummy"],
+        "romantic": ["Vanilla Sky", "Meet Joe Black"],
+        "drama": ["Meet Joe Black"],
+        "thriller": ["Vanilla Sky"],
+        "action": ["Mission Impossible"],
+        }
+    genres_keys = list(genres_data.keys())
+    print(f'{">>> Available Genres: "} {genres_keys}')
+    dict_values = genres_data.values()
+    genre = input("Enter specific genre: ").strip()
+    if genre in genres_keys:
+        dict_values = genres_data[genre]
+        print(f'>>> Available movies: {list(dict_values)}')
+        lst = list(dict_values)
+        new_list = []
+        movie = input("Enter movie: ").strip()
+        for x in lst:
```

Comment:
`for movie in movies`
there is no need to convert dict.values() to list if you only want to do for loop

Reply:
actually why not just check `movie in movies`

Reply:
if you do loop to lower case input value then it probably needs to be:
`if movie.lower() == x.lower()`, right? Or this is sort of smart search?)